cpp/velox/shuffle/VeloxSortShuffleWriter.cc

```diff
@@ -106,8 +104,17 @@ arrow::Status VeloxSortShuffleWriter::init() {
       options_.partitioning == Partitioning::kSingle,
       arrow::Status::Invalid("VeloxSortShuffleWriter doesn't support single partition."));
   allocateMinimalArray();
-  sortedBuffer_ = facebook::velox::AlignedBuffer::allocate<char>(kSortedBufferSize, veloxPool_.get());
-  rawBuffer_ = sortedBuffer_->asMutable<uint8_t>();
+  // In Spark, sortedBuffer_ memory and compressionBuffer_ memory are pre-allocated and counted into executor
+  // memory overhead. To align with Spark, we use arrow::default_memory_pool() to avoid counting these memory in Gluten.
```

Comment:
@FelixYBW arrow::default_memory_pool  is used to allocate the sort buffer and compress buffer.

Reply:
@zhztheplayer Can you help look at here? Thanks!

Reply:
Do we need to add a function `defaultArrowMemoryPool` to `VeloxMemoryManager` to unify the memory pool usage?

Reply:
I think the code looks fine now as we don't have a mechanism to count global allocation of Arrow into Spark overhead memory. 

In future we may report both Arrow and Velox's global pool usages to one counter which requires for some designs. So far we don't have that.